src/index/datastore.ts

```diff
@@ -188,8 +188,8 @@ export class Datastore {
         }
 
         // All fields on an object.
-        if (object.$types.contains(FIELDBEARING_TYPE) && "$fields" in object) {
-            for (const field of object.$fields as Iterable<Field>) {
+        if (object.$types.contains(FIELDBEARING_TYPE) && "fields" in object) {
```

Comment:
This is broken, is it not? I'm pretty sure it should be `$fields`.

Reply:
https://github.com/blacksmithgu/datacore/blob/d75046c5bfe97bbbde503aeeef720658ccab97b3/src/expression/field.ts#L26-L31

here, it's `fields`, not `$fields`.